### PR TITLE
Adds/Changes various machine descriptions

### PR DESCRIPTION
**Pull request by theselfish**
# About the pull request

I dislike wordless or poorly written item descriptions. It saddens me to see them.

# Explain why it's good for the game

Having a description is good.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog
:cl: Cursor
spellcheck: Changed Chem Dispenser to Chemical Dispenser, added descriptions to the cryo cell, rail computer and chemical dispenser. Updated the descriptions for the Blood Dispenser and Dropship computer. 
/:cl:

### CODE REVIEW

File: code/game/machinery/cryo.dm [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 /obj/structure/machinery/cryo_cell
 	name = "cryo cell"
+	desc = "A donation from the old A.W project, using cryogenic technology, it slowly heals whomever is inside the tube."
```
Comment (BeagleGaming1) [inline code/game/machinery/cryo.dm:5]:
```suggestion
	desc = "A donation from the old A.W. project, using cryogenic technology. It slowly heals whoever is inside the tube."
```

File: code/game/machinery/vending/vendor_types/medical.dm [hunk 2]
```diff
@@ -272,7 +272,7 @@
 
 /obj/structure/machinery/cm_vending/sorted/medical/blood
 	name = "\improper MM Blood Dispenser"
-	desc = "Marine Med brand Blood Pack Dispensary"
+	desc = "The Marine Med Brand Blood Pack Dispensary is the premier, top of the line blood dispenser of 2105! Get yours today!" //Don't update this year, the joke is it's old.
```
Comment (BeagleGaming1) [inline code/game/machinery/vending/vendor_types/medical.dm:275]:
```suggestion
	desc = "The Marine Med Brand Blood Pack Dispensary is the premier, top-of-the-line blood dispenser of 2105! Get yours today!" //Don't update this year, the joke is it's old.
```

File: code/modules/shuttle/computers/dropship_computer.dm [hunk 3]
```diff
@@ -1,6 +1,6 @@
 /obj/structure/machinery/computer/shuttle/dropship/flight
 	name = "dropship navigation computer"
-	desc = "flight computer for dropship"
+	desc = "A flight computer than can be used for autopilot or long range flights."
```
Comment (BeagleGaming1) [inline code/modules/shuttle/computers/dropship_computer.dm:3]:
```suggestion
	desc = "A flight computer that can be used for autopilot or long-range flights."
```

**Review by BeagleGaming1 — COMMENTED** [review]
Minor spelling suggestions

**Comment by theselfish** [comment]
@harryob @BeagleGaming1 Sorry, never noticed I had comments until now.

Suggestions merged.

**Comment by BeagleGaming1** [comment]
> Suggestions merged.

You gotta undraft it

**Comment by theselfish** [comment]
> > Suggestions merged.
> 
> You gotta undraft it

It was Harry who marked it as draft. I don't believe I can unmark it...

**Review by BeagleGaming1 — APPROVED** [review]

**Pull request merged**